### ISSUE TITLE
[BUG]: Pycaret 3.0 Feature "Label" doesn't show up on plotly trendline.

### ISSUE THREAD

**Issue by residual-07** [pycaret/pycaret#3416]
### pycaret version checks

- [X] I have checked that this issue has not already been reported [here](https://github.com/pycaret/pycaret/issues).

- [X] I have confirmed this bug exists on the [latest version](https://github.com/pycaret/pycaret/releases) of pycaret.

- [ ] I have confirmed this bug exists on the master branch of pycaret (pip install -U git+https://github.com/pycaret/pycaret.git@master).

### Issue Description

I've noticed the strange issue when i want to plot predictions with plotly. 

In previous version of pycaret, i haven't had problem, but in RC releases i've seen that it doesnt allow and it throws me a value error. I haven't change the code, since i was using it from previous version, and now it doesnt work. 

### Reproducible Example

```python
# generate predictions on the original dataset
predictions = predict_model(lgbm1, data=df)

# add a date column in the dataset
predictions['Dates'] = pd.date_range(start='2021-01-01 00:00:00', end = '2022-05-14 21:00:00', freq = 'H')

# line plot
fig = px.line(predictions, x='Dates', y=["target", "Label"], template = 'plotly_dark')

# add a vertical rectange for test-set separation
fig.add_vrect(x0="2022-05-01", x1="2022-05-14", fillcolor="grey", opacity=0.30, line_width=0)

fig.show()
```

### Expected Behavior

![newplot (2)](https://user-images.githubusercontent.com/124274176/226315652-4f65deca-fb65-4776-b376-aa1aa327ab5d.png)

### Actual Results

```python-traceback
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
/var/folders/4v/s07q12hn4vs2j8_v4_wq30jc0000gn/T/ipykernel_25206/2458448071.py in <cell line: 8>()
      6 
      7 # line plot
----> 8 fig = px.line(predictions, x='Dates', y=["MIPS", "Label"], template = 'plotly_dark')
      9 
     10 # add a vertical rectange for test-set separation

/usr/local/anaconda3/envs/pycaret_3_testing/lib/python3.8/site-packages/plotly/express/_chart_types.py in line(data_frame, x, y, line_group, color, line_dash, symbol, hover_name, hover_data, custom_data, text, facet_row, facet_col, facet_col_wrap, facet_row_spacing, facet_col_spacing, error_x, error_x_minus, error_y, error_y_minus, animation_frame, animation_group, category_orders, labels, orientation, color_discrete_sequence, color_discrete_map, line_dash_sequence, line_dash_map, symbol_sequence, symbol_map, markers, log_x, log_y, range_x, range_y, line_shape, render_mode, title, template, width, height)
    262     a polyline mark in 2D space.
    263     """
--> 264     return make_figure(args=locals(), constructor=go.Scatter)
    265 
    266 

/usr/local/anaconda3/envs/pycaret_3_testing/lib/python3.8/site-packages/plotly/express/_core.py in make_figure(args, constructor, trace_patch, layout_patch)
   1989     apply_default_cascade(args)
   1990 
-> 1991     args = build_dataframe(args, constructor)
   1992     if constructor in [go.Treemap, go.Sunburst, go.Icicle] and args["path"] is not None:
   1993         args = process_dataframe_hierarchy(args)

/usr/local/anaconda3/envs/pycaret_3_testing/lib/python3.8/site-packages/plotly/express/_core.py in build_dataframe(args, constructor)
   1404     # now that things have been prepped, we do the systematic rewriting of `args`
   1405 
-> 1406     df_output, wide_id_vars = process_args_into_dataframe(
   1407         args, wide_mode, var_name, value_name
   1408     )

/usr/local/anaconda3/envs/pycaret_3_testing/lib/python3.8/site-packages/plotly/express/_core.py in process_args_into_dataframe(args, wide_mode, var_name, value_name)
   1244 
   1245                 if length and len(argument) != length:
-> 1246                     raise ValueError(
   1247                         "All arguments should have the same length. "
   1248                         "The length of argument `%s` is %d, whereas the "

ValueError: All arguments should have the same length. The length of argument `y` is 2, whereas the length of  previously-processed arguments ['Dates'] is 11974
```

### Installed Versions

<details>
System:
    python: 3.8.13 (default, Mar 28 2022, 06:16:26)  [Clang 12.0.0 ]
executable: /usr/local/anaconda3/envs/pycaret_3_testing/bin/python
   machine: macOS-10.16-x86_64-i386-64bit

PyCaret required dependencies:
                 pip: 21.2.4
          setuptools: 61.2.0
             pycaret: 3.0.0
             IPython: 7.34.0
          ipywidgets: 7.7.2
                tqdm: 4.64.0
               numpy: 1.21.6
              pandas: 1.4.2
              jinja2: 3.0.3
               scipy: 1.8.1
              joblib: 1.2.0
             sklearn: 1.1.1
                pyod: 1.0.9
            imblearn: 0.9.1
   category_encoders: 2.5.0
            lightgbm: 3.3.2
               numba: 0.55.2
            requests: 2.27.1
          matplotlib: 3.5.2
          scikitplot: 0.3.7
         yellowbrick: 1.4
              plotly: 5.13.1
             kaleido: 0.2.1
         statsmodels: 0.13.5
              sktime: 0.16.1
               tbats: 1.1.0
            pmdarima: 1.8.5
              psutil: 5.9.1

PyCaret optional dependencies:
                shap: 0.41.0
           interpret: Not installed
                umap: 0.5.3
    pandas_profiling: 3.6.6
  explainerdashboard: 0.4.1.1
             autoviz: Not installed
           fairlearn: Not installed
             xgboost: Not installed
            catboost: Not installed
              kmodes: Not installed
             mlxtend: Not installed
       statsforecast: Not installed
        tune_sklearn: Not installed
                 ray: Not installed
            hyperopt: Not installed
              optuna: 3.1.0
               skopt: Not installed
              mlflow: Not installed
              gradio: 3.8.1
             fastapi: 0.85.1
             uvicorn: 0.19.0
              m2cgen: Not installed
           evidently: Not installed
               fugue: Not installed
           streamlit: Not installed
             prophet: Not installed
</details>

**Comment by moezali1** [comment]
@residual-07 This doesn't relate to PyCaret.

The error message you are seeing indicates that there is a mismatch in the length of the input arguments for a function. Specifically, it seems that you are trying to pass two arguments to a function, where one of the arguments, y, has length 2, but the other argument, Dates, has length 11974.

**Comment by residual-07** [comment]
Thats right, but I'm using completely same line of code which actually worked in previous release. I haven't encounter that problem in earlier version, but this recent one is throws me missmatch length. So, i've tried with troubleshooting of plotly and it i didnt have any progress so i concluded that i dont have problem with plotly. However, when i tried older version of pycaret, i was getting no error in missmatch.
